### PR TITLE
Feature/read ft category implementation

### PR DESCRIPTION
**Pull request by MackowiakPiotr**

### CODE REVIEW

**Review by odzio33 — CHANGES_REQUESTED** [review]
Change required. 
Small code fixes with entity joins and break line in the right place.

**Review by odzio33 — APPROVED** [review]
Looks good to me

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java [hunk 1]
```diff
@@ -50,6 +49,9 @@ public FinancialTransactionDTO createFinancialTransaction(@Valid FinancialTransa
 
     @Override
     public List<FinancialTransactionDTO> getFinancialTransactionsByWalletId(@Min(1) @NotNull Long walletId) {
+        if (!walletRepository.existsById(walletId)){
```
Comment (dawciobiel) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java:52]: How it should be formatted:
`if (!walletRepository.existsById(walletId)){`
or
`if (!walletRepository.existsById(walletId)) {`
?

**Review by dawciobiel — APPROVED** [review]
I think this looks good to me. 
Remember to properly format the code before the commit & push are done.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java (hunk 1 above)
Comment (MackowiakPiotr) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java:52]: You'd have to ask the author of the method

**Pull request merged**